pages/linux/dconf-write.md

```diff
@@ -0,0 +1,24 @@
+# dconf write
+
+> Write a value to a dconf database path.
```

Comment:
```suggestion
> Write a value to a dconf database path.
> See also `dconf`.
```

Reply:
I agree with referencing `dconf-write` in `dconf`, but isn't this one implied?

Especially with the rule of:

> You should always add a base page

Reply:
Yeah, it is implied 🤔

Reply:
Resolve?

Reply:
> Resolve?

I'd wait for @marchersimon to respond, but I think so